src/pages/beta/[viewType]/programmes/[programmeSlug]/units/[unitSlug]/lessons/[lessonSlug].tsx

```diff
@@ -244,7 +246,7 @@ const LessonOverviewPage: NextPage<LessonOverviewPageProps> = ({
           </Box>
         ) : (
           <>
-            <Grid $pt={[48]}>
+            <Grid id="test" $pt={[48]}>
```

Comment:
Is this used anywhere? If so, could we give it a unique id?

Reply:
whoops. I'll remove that